### PR TITLE
BAPL-314: fix the comments of pull request #2180

### PR DESCRIPTION
**Pull request by xiezhang7**

### CODE REVIEW

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser3** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by romartin** [comment]
ok to test

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/canvas/controls/containment/CaseManagementContainmentAcceptorControlImpl.java [hunk 1]
```diff
@@ -236,12 +237,13 @@ int getAddIndex(final WiresShape wiresShape,
...
-                    return parent.getOutEdges().size() - 1;
-                } else {
-                    return parent.getOutEdges().size();
-                }
+            }
+
+            // Otherwise, add to a stage vertically
+            if (state.getOriginalParent().isPresent() && state.getOriginalParent().get().equals(container)) {  // same stage
+                return parent.getOutEdges().size() - 1;
+            } else {
```
Comment (LuboTerifaj) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/canvas/controls/containment/CaseManagementContainmentAcceptorControlImpl.java]: Else statement here can be removed as well.

**Comment by xiezhang7** [comment]
Hi @LuboTerifaj , I have corrected the code. Thanks.

**Comment by xiezhang7** [comment]
Jenkins execute full downstream build

**Comment by LuboTerifaj** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Jenkins retest this

**Comment by xiezhang7** [comment]
Rebased to latest master branch to fix the build

Jenkins execute full downstream build

**Comment by xiezhang7** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Pull request merged**